Resources/Prototypes/Floof/Interactions/mood_interactions.yml

```
@@ -0,0 +1,31 @@
+- type: Interaction
+  id: Kiss
+  parent: [BaseGlobal, BaseLips]
+  priority: 0
+  delay: 0.4
+  range: {max: 1}
+  hideByRequirement: true
+  requirement:
+    !type:MobStateRequirement
+    inverted: true
```

Lesser priority, inlined effect
```suggestion
  parent: [BaseGlobal, BaseHands]
  priority: -1
  delay: 0.4
  range: {max: 1}
  hideByRequirement: true
  requirement:
    !type:MobStateRequirement
    inverted: true
  effectSuccess:
    popup: Obvious
    sound: {path: /Audio/Floof/Lewd/kiss.ogg}
```